modules/core/src/test/scala/FuncSuite.scala

```diff
@@ -30,12 +30,19 @@ import io.blindnet.pce.api.*
 import cats.data.Kleisli
 import io.blindnet.identityclient.IdentityClientBuilder
 import io.blindnet.identityclient.auth.*
+import io.circe.literal.*
+import org.http4s.circe.*
+import io.blindnet.identityclient.IdentityClient
+import io.blindnet.jwt.*
 
 trait FuncSuite extends IOSuite {
 
   val appId = UUID.fromString("6f083c15-4ada-4671-a6d1-c671bc9105dc")
   val ds    = DataSubject("fdfc95a6-8fd8-4581-91f7-b3d236a6a10e", appId)
 
+  val key      = TokenPrivateKey.fromString("m+UYVxv2NTnqVz8i1J6BnJ7aZshNu9SqfPmaIMv2nwM=")
```

Comment:
You can use `TokenPrivateKey.generateRandom()` instead of an hardcoded key, and then `.toPublicKey().toString()` to get the string representation of the public key if needed

Reply:
Done, thanks